### PR TITLE
Minor CSS updates

### PR DESCRIPTION
**Pull request by yuliia-biletska**

### CODE REVIEW

**Review by vasyl-krupa — APPROVED** [review]
Looks correct

File: plugins/rocks.xml.webhelp/css/layout-common.css [hunk 1]
```diff
@@ -110,6 +110,6 @@ main {
 .footer-text {
     padding: 1rem;
     text-decoration: none;
-    color: #1a1e21;
+    color: #FFFFFF;
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/css/layout-common.css]: Text color should be #b2b2b2 as far as I remember

Comment (yuliia-biletska) [reply plugins/rocks.xml.webhelp/css/layout-common.css]: fixed

**Review by ibarinova — APPROVED** [review]
Approved

**Pull request merged**